### PR TITLE
fix: Update config for secret using Docker Hub

### PR DESCRIPTION
**Pull request by ram4444**

### CODE REVIEW

File: content/en/docs/reference/components/docker-registry.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ kubectl create secret generic jenkins-docker-cfg --from-file=./config.json
 
 ## Using Docker Hub
 
-If you want to publish images to docker hub then you need to modify your `config.json` as described above to something like:
+If you want to publish images to docker hub then you need to modify your `config.json` in your home directory as described above to something like:
```
Comment (hferentschik) [inline content/en/docs/reference/components/docker-registry.md:82]: in your home directory? That seems wrong. I guess you are referring to "~/.docker/config.json", right?

Comment (ram4444) [reply content/en/docs/reference/components/docker-registry.md:82]: yes you are correct

Comment (hferentschik) [reply content/en/docs/reference/components/docker-registry.md:82]: do you mind updating the PR?

Comment (ram4444) [reply content/en/docs/reference/components/docker-registry.md]: done pls check, thx for your remind

**Comment by hferentschik** [comment]
Hi @ram4444, thanks for the pull request. When reviewing the whole page I found the whole page could need an overhaul. I created pull request #2645 to supersede your changes. I retained your commit, so your contribution is not lost ;-)

Have a look at the pull request and let me know what you think.

**Comment by hferentschik** [comment]
Closing this pull request in favour of #2645

**Pull request closed without merging**